kiwi/volume_manager/btrfs.py

```diff
@@ -287,6 +287,21 @@ def umount_volumes(self):
 
         return all_volumes_umounted
 
+    def get_mountpoint(self) -> str:
+        """
+        Provides btrfs root mount point directory
+
+        Effective use of the directory is guaranteed only after sync_data
+
+        :return: directory path name
+
+        :rtype: string
+        """
+        sync_target: [list][str] = [self.mountpoint, '@']
```

Comment:
[I would not use](https://www.python.org/dev/peps/pep-0484/#type-comments) direct built-in for lists. This is an old way but after 3.5 was out, this is gone. And if you will use MyPy, it will work better.

```python
from typing import List

sync_target: [List][str] ....
```

Reply:
for sure a better read, thanks